quantum/quantum.h

```diff
@@ -154,7 +154,7 @@ extern layer_state_t layer_state;
 #if defined(__AVR__)
 typedef uint8_t pin_t;
 
-#    define setPinInput(pin) (DDRx_ADDRESS(pin) &= ~_BV((pin)&0xF))
+#    define setPinInput(pin) (DDRx_ADDRESS(pin) &= ~_BV((pin)&0xF), PORTx_ADDRESS(pin) &= ~_BV((pin)&0xF)
```

Comment:
Did ... you by chance mean: 
```suggestion
#    define setPinInput(pin) (DDRx_ADDRESS(pin) &= ~_BV((pin)&0xF); PORTx_ADDRESS(pin) &= ~_BV((pin)&0xF)
```

Reply:
Actually, this should be a comma.

Reply:
Okay, I wasn't sure about that.

Reply:
However, this seems to be causing this:
https://travis-ci.org/qmk/qmk_firmware/builds/587884989#L924

Reply:
There's a missing closing parenthesis at the end of the macro definition. I'll submit a fix suggestion.